eng/pipelines/coreclr/templates/perf-job.yml

```diff
@@ -286,12 +267,3 @@ jobs:
     - script: $(Build.SourcesDirectory)/src/tests/build$(scriptExt) $(buildConfig) $(archType) generatelayoutonly $(librariesOverrideArg) $(_crossBuildPropertyArg)
       displayName: Create Core_Root
       condition: and(succeeded(), ne(variables.runtimeFlavorName, 'Mono'), ne('${{ parameters.runtimeType }}', 'wasm'), not(in('${{ parameters.runtimeType }}', 'AndroidMono', 'iOSMono', 'iOSNativeAOT')))
-
-    # Copy the runtime directory into the testhost folder to include OOBs.
```

Comment:
The removal of these steps breaks our mono runs. We expect to have a runnable mono version in .dotnet-mono, and it needs to have the files from the runtime, testhost, and `corerun` combined there. We don't need the build step part of this anymore as we get all of these outputs in the unzipped build.